src/platforms/common/performance/index.mdx

```diff
@@ -134,6 +135,23 @@ If you are using `react-router`, we provide <PlatformLink to="/configuration/int
 
 </PlatformSection>
 
+<PlatformSection supported={["php"]}>
+
+## Improve Response Time
+
+Using the performance capabilities of PHP has some impact on your response time (depending on the `traces_sample_rate` you configured).
+Because of the nature of PHP, requests in most cases are sent in the same process as you serve your users' response.
```

Comment:
```suggestion
Because of the nature of PHP, in most cases, requests are sent in the same process as the one in which you serve your users' response.
```